app/code/community/HE/TwoFactorAuth/etc/config.xml

```diff
@@ -75,7 +96,6 @@
                     </he_twofactorauth_observer>
                 </observers>
             </admin_session_user_login_success>
-
```

Comment:
Let's not change this indentation for no reason. Force push this out, I don't want it in the `git blame`.